src/main/java/uk/ac/ebi/ena/backend/service/EnaPortalService.java

```diff
@@ -335,4 +356,50 @@ public void sendEmail(String recipientEmail, String message, String subject, Str
         console.info("Email successfully sent to:{}", recipientEmail);
 
     }
+
+    public boolean authenticateUser(AuthenticationDetail authenticationDetail) {
+
+        String userName = authenticationDetail.getUserName();
+        String password = authenticationDetail.getPassword();
+
+        if (StringUtils.isNotBlank(userName) && StringUtils.isNotBlank(password)) {
+
+            String portalAPIAuthEndpoint = Constants.PORTAL_API_EP + "/auth?dataPortal=" +
+                    CommonUtils.getDataPortalId(userName);
+            log.info("portalAPIAuthEndpoint: " + portalAPIAuthEndpoint);
+
+            HttpHeaders httpHeaders = new HttpHeaders();
+            httpHeaders.add("Accept", Constants.APPLICATION_JSON);
+            httpHeaders.setBasicAuth(userName, password);
+
+            restTemplate.getInterceptors().add(new BasicAuthorizationInterceptor(userName, password));
```

Comment:
BasicAuthorizationInterceptor is deprecated use BasicAuthenticationInterceptor instead

Reply:
Removed BasicAuthorizationInterceptor and now passing basicAuth param to header.